sdk/webpubsub/azure-messaging-webpubsubservice/examples/send_messages_aad_apim_proxy.py

```diff
@@ -0,0 +1,49 @@
+import io
+import logging
+import os
+
+from azure.messaging.webpubsubservice import WebPubSubServiceClient
+from azure.identity import DefaultAzureCredential
+from azure.core.exceptions import HttpResponseError
+
+logging.basicConfig(level=logging.DEBUG)
+LOG = logging.getLogger()
+
+# Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+# AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET, WEBPUBSUB_ENDPOINT
+try:
+    endpoint = os.environ.get("WEBPUBSUB_ENDPOINT")
```

Comment:
I think because you're calling `.get`, we will never enter the `KeyError` portion. So I think it would be better to do `os.environ["WEBPUBSUB_REVERSE_PROXY_ENDPOINT"]` instead (note: i think the env var `WEBPUBSUB_REVERSE_RPOXY_ENDPOINT` has a typo)